lessons/AI/ai-statistical-lang-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,452 @@
+
+The very same process that generates "hallucinatory" text _also_ generates the "non-hallucinatory" text. This truth helps us to understand _why_ it is so difficult to *fix* the "hallucination" problem.
+
+This term also attributes intent and consciousness to the AI, giving it human qualities when it is merely executing a program exactly as it is intended to do.
+}
+
+
+
+=== Synthesize
+
+@QandA{
+
+Critics of ChatGPT and other language models raise a variety of concerns. Consider each of them, below.
+
+@Q{This form of AI often has trouble with _facts_. Put another way, ChatGPT sometimes "makes stuff up." Why does this happen? What is actually going on?}
+
+@A{When ChatGPT produces false or misleading information, it is not glitching nor is there a bug. ChatGPT is just doing what it does, following the model as it ought to.}
+
+
+@Q{Others complain that ChatGPT has biases that can be seen in its text output. Where do these biases come from?}
+
+@A{If there are biases in the corpus, there will likely be biases in the output.}
+}
+
+
+
+
+
+== What is the Role of Language?
+
+=== Overview
+
+Students consider whether statistical language modeling requires language.
+
+=== Launch
+
+
+@lesson-instruction{
+- Play a game of tic-tac-toe with your neighbor.
+- Play the tic-tac-toe game spelled out (Document A, on the right) with your neighbor.
+@right{@image{images/docA.png, 100}}
+
+- Not sure how to proceed?
+** The tic-tac-toe board is a 3x3 coordinate plane.
+** Each row of the document contains two pieces of information: (1) the player (X or O), and (2) the ordered pair for the selected cell.
+}
+
+
+=== Investigate
+
+You've used a paper, pencil, and probability to apply the principles of statistical language modeling. It's time to peek behind the curtain and see how a computer can put this model to use! To make that happen, we're going to explore Soekia, a simplified text generation tool designed for student learning. As we explore, we are going to consider: What is the role of _language_ in a statistical language model?
+
+@lesson-instruction{
+- Go to @link{http://bootstrapworld.org/SoekiaGPT/}
+- Click the "LOOK INSIDE" button at the top right of your screen.
+- Scroll to the right until the green, right-most panel ("Documents") is in view.
+- Click the "Collections" icon in the upper right corner.
+- From the drop-down menu that appears, select "tic-tac-toe".
+- Complete @printable-exercise{tic-tac-toe.adoc}
+}
+
+@teacher{
+Invite students to share their discoveries. We encourage you to allow students the opportunity to play with Soekia. There are many possible discoveries awaiting your students!}
+
+=== Synthesize
+
+@QandA{
+
+@Q{Does a statistical language model require language? How do you know?}
+
+@A{No. It just needs some sort of corpus. Natural language is absolutely not needed. Chess games, musical compositions, and tic-tac-toe games are all adequate!}
```

Comment:
Noting that I also don't know what natural language means, so this isn't clearer.